auth-jwt-service/README.md

```diff
@@ -71,6 +71,8 @@ CREATE TABLE IF NOT EXISTS store.shopping_cart (userid text PRIMARY KEY, item_co
 
 INSERT INTO store.shopping_cart (userid, item_count, last_update_timestamp) VALUES ('9876', 2, toTimeStamp(toDate(now())));
 INSERT INTO store.shopping_cart (userid, item_count, last_update_timestamp) VALUES ('1234', 5, toTimeStamp(toDate(now())));
+GRANT MODIFY ON TABLE store.shopping_cart TO web_user ;
```

Comment:
^ github markdown supports the `cql` as the code block tag. It results in the same highlighting, but `sql` is a bit confusing when looking at the raw markdown. Example: https://gist.github.com/mpenick/7a0e82affa51477b97558c18e8f0d6fd

Reply:
Neat, I didn't realize that.